src/rebuild/srv.c

```diff
@@ -1,5 +1,5 @@
 /**
- * (C) Copyright 2016-2024 Intel Corporation.
+ * (C) Copyright 2016-2025 Intel Corporation.
```

Comment:
[Nit] Should add an HPE copyright line instead. A Git config problem?

Reply:
See HPE copyright line has been existed already, updating intel copyright data is done by git hook scripts automatically, it might be a bug of script or somehow my hooks is out  of date. @daltonbohning any idea?

Reply:
See HPE copyright line has been existed already, updating intel copyright data is done by git hook scripts automatically, it might be a bug of script or somehow my hooks is out  of date. @daltonbohning any idea?

Reply:
If you have the hooks installed with `git config core.hookspath utils/githooks` then they won't be out of date - they will always come from the current versions on the branch. Did you commit with an intel email at some point and then rebase? The githook looks at the signed off email to determine whether to use HPE or Intel

Reply:
Thanks, i checked my local configurations, it is fine. maybe because i commited changes in the VM which is still intel email address, anyway, i revert this change.